includes/definitions.inc.php

```diff
@@ -1363,7 +1363,7 @@
 $config['os'][$os]['icon'] = 'hikvision';
 
 // Canopy / Cambium support
-$os = 'canopy';
```

Comment:
Please keep in mind $os is for naming the OS that a device runs not the company or product range. So for cisco we have ios, iosxe, etc.